sdk/liftrqumulo/Azure.ResourceManager.LiftrQumulo/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Please provide the expected release date of the first beta, 2 weeks buffer at least